app/models/health/release_saver.rb

```diff
@@ -44,7 +44,7 @@ def update
         source_type: @form.class.name,
         user_id: @user.id,
         user_full_name: @user.name_with_email,
-        date_of_activity: Date.current,
+        date_of_activity: [@form.signature_on, @form.participation_signature_on].max,
```

Comment:
Does this need an `|| Date.current`?

Reply:
This should only be triggered if both are set, and the specific issue was that the QA should match the signature dates. They had back-dated an ROI and wanted the QA back-dated as well.